src/main/java/org/folio/rest/impl/OrdersImpl.java

```diff
@@ -105,28 +106,34 @@ public void postOrdersCompositeOrders(String lang, CompositePurchaseOrder compPO
 
     final HttpClientInterface httpClient = AbstractHelper.getHttpClient(okapiHeaders);
     PostOrdersHelper helper = new PostOrdersHelper(httpClient, okapiHeaders, asyncResultHandler, vertxContext, lang);
-    loadConfiguration(okapiHeaders, vertxContext, logger).thenAccept(config -> {
-      validatePoLinesQuantity(compPO, config);
-      logger.info("Creating PO and POLines...");
-      helper.createPurchaseOrder(compPO)
-        .thenAccept(withIds -> {
-
-          logger.info("Applying Funds...");
-          helper.applyFunds(withIds)
-            .thenAccept(withFunds -> {
-
-              logger.info("Successfully Placed Order: " + JsonObject.mapFrom(withFunds).encodePrettily());
-              httpClient.closeClient();
-              javax.ws.rs.core.Response response = PostOrdersCompositeOrdersResponse.respond201WithApplicationJson(withFunds,
-                PostOrdersCompositeOrdersResponse.headersFor201().withLocation(String.format(ORDERS_LOCATION_PREFIX, withFunds.getId())));
-              AsyncResult<javax.ws.rs.core.Response> result = Future.succeededFuture(response);
-              asyncResultHandler.handle(result);
+
+    validate(lang, okapiHeaders, vertxContext, httpClient, compPO).thenAccept(errs -> {
```

Comment:
This check was added basically for cases when an order is being created by external modules (like GOBI) because UI will work only with active vendors.
@craigmcnally @KVupp - is it okay to reject order creation in this case or you think order still can be created but in `Pending` state? At later stage we might add alert for such order specifying the reason...

Reply:
Checking that the vendor actually exists is probably not needed here... mod-gobi performs a lookup of the vendor UUID given a vendor code, e.g. GOBI

Reply:
Okay, this extra validation then can be removed if modules like `mod-gobi` are going to validate the vendor as well (i.e. vendor exists and active).

Reply:
validating that all UUID's provided are valid is probably something we can hold off on for now